lenovo/thinkpad/p16s/amd/gen1/default.nix

```diff
@@ -1,6 +1,13 @@
 { config, lib, pkgs, ... }:
 
 {
+
+  imports = [
+    ../../../../../common/cpu/amd
+    ../../../../../common/gpu/amd
+    ../../../../../common/pc/laptop/acpi_call.nix
+  ];
+
   # For mainline support of rtw89 wireless networking
-  boot.kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "5.16") pkgs.linuxPackages_latest;
+  boot.kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "5.16") pkgs.linuxPackages_6_1;
```

Comment:
Why is this change needed? `linuxPackages_latest.kernel` already points 6.1:

```
nix-repl> linuxPackages_latest.kernel
«derivation /nix/store/fcs136ak5c61ax5a12hw111zhy06l78w-linux-6.1.drv»
```

if there is no `linuxPackages_6_1` than evaluation fails (this can happen if the kernel is for example removed from nixpkgs)